### PR TITLE
feat(components): add mutation comparison with table

### LINKED ISSUE #77: Mutation comparison - Table: Add a table view that, for each mutation, shows the prevalence in the different variants

### PR DESCRIPTION
**Pull request by JonasKellerer**
resolves #77 

### Summary

This adds a new component, which shows a mutation comparison of variations.
Currently, there is only a table view.

### Screenshot

grafik

### PR Checklist

- [x] The implemented feature is covered by an appropriate test.

### CODE REVIEW

File: components/src/preact/mutationComparison/mutation-comparison.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+                lapisFilter: {
+                    country: 'Switzerland',
+                    pangoLineage: 'B.1.1.7',
+                    dateFrom: dateFromOtherVariant,
+                    dateTo: dateToOtherVariant,
+                },
+            },
+        ],
+        sequenceType: 'nucleotide',
+        views: ['table'],
```
Comment (fengelniederhammer) [inline components/src/preact/mutationComparison/mutation-comparison.stories.tsx:63]: `meta` already has `views` set.

File: components/src/preact-components/display/mutation-comparison-component.stories.ts [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+                lapisFilter: {
+                    country: 'Switzerland',
+                    pangoLineage: 'B.1.1.7',
+                    dateFrom,
+                    dateTo,
+                },
+            },
+        ],
+        sequenceType: 'nucleotide',
+        views: ['table'],
```
Comment (fengelniederhammer) [inline components/src/preact-components/display/mutation-comparison-component.stories.ts:66]: Same here, I think we don't need it here, if `meta` already has it.

**Review by fengelniederhammer — APPROVED** [review]

**Review by chaoran-chen — COMMENTED** [review]
I just looked at the screenshot and it's great!

One improvement for all tables: #91

File: components/src/preact/mutationComparison/mutation-comparison.stories.tsx (hunk 1 above)
Comment (JonasKellerer) [reply components/src/preact/mutationComparison/mutation-comparison.stories.tsx:63]: The views from meta is only to provide information, what can be selected in the controls. It does not set the value. One has to provide it as an arg.

**Pull request merged**